examples/amp-story/quiz.html

```diff
@@ -69,6 +69,47 @@ <h2>True or False: Cats can be right-pawed or left-pawed.</h2>
           </amp-story-quiz>
         </amp-story-grid-layer>
       </amp-story-page>
+      <amp-story-page id="quiz-question-3">
+        <amp-story-grid-layer template="fill">
+            <amp-img src="https://i.pinimg.com/originals/8d/b1/e9/8db1e9b7cc3ee52b13cabe3a53c62f05.jpg"
+            width="720" height="1280"
+            layout="responsive">
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical">
+          <style>
+            #cat-question-cute {
+              position: absolute;
+              top: 600px;
+              left: 100px;
+              width: 40px;
+              height: 20px;
+            }
+          </style>
+          <amp-story-quiz
+            id='cat-question-cute'
+            layout="flex-item">
```

Comment:
Is this required?

Reply:
When I remove the layout attribute I get an error:

```
Uncaught Error: The element did not specify a layout attribute. Check https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout and the respective element documentation for details.​​​
```

This is because when the width and height attributes are not present, the element defaults to `layout: container`, which is unsupported - for now I've gone ahead and added `container` to the supported layouts, but we may need to revisit this and consider if directly specifying `layout="flex-item"` is more desired.